src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,304 @@
+package duke;
+
+import duke.fileHandling.FileHandler;
+import duke.task.*;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Duke {
+    /**
+     * A divider (horizontal line).
+     */
+    public static final String DIVIDER_LINE_ONLY = "_____________________________________________________________";
+    /**
+     * A divider (horizontal line) with line break at the end.
+     */
+    public static final String DIVIDER = "_____________________________________________________________\n";
+
+    /**
+     * Prints greetings when the program starts.
+     */
```

Comment:
Great that you're using comments to explain your code! For short single-line comments, using a double-slash (// ... ) would be sufficient.